app/src/components/results/layoutComposer.js

```diff
@@ -155,7 +155,7 @@ const intPresenter = (uid, name, val) => {
 };
 
 const jsonPresenter = (uid, name, val) => {
-  // we only shows 2 key pairs
+  // we only shows 3 key pairs
```

Comment:
still might be valuable to explicitly tie the 3 in this comment to the value `2` in code below. since one might not immediately recognize that in the future

and/or maybe move this comment to line 172 nearer the mechanism that stops at 3